minigotchi-ESP32/config.h

```diff
@@ -6,6 +6,7 @@
 #define CONFIG_H
 
 #include <Arduino.h>
+#include <TFT_eSPI.h>
```

Comment:
i don't think we need that here, it's all in `display.*`

Reply:
Got it.